style.css

```diff
@@ -0,0 +1,18 @@
+h2 {
+  color: #000000;
+  font-family: "Open Sans", Arial, sans-serif;
+}
```

Comment:
I think it would be better to use classes than styling the HTML tags directly. I think it would be better to use classes than styling the HTML tags directly. HTML and CSS design best practices recommend giving an element a class rather than directly using its tag name.

Reply:
- [ ] Great job pointing out an issue with your CSS selector! 🎨  However, saying that HTML and CSS best practices recommend this is too vague of a reason. 🤓 Why this change should be implemented and how the code author could benefit from using classes instead of element selectors? Please, consider answering these questions as straightforwardly as possible and editing this comment! 💬 Part of giving a good review is pointing out a clear reason to make the change. 🤖 🛠️